Lib/fontbakery/profiles/glyf.py

```diff
@@ -127,3 +127,36 @@ def com_google_fonts_check_glyf_non_transformed_duplicate_components(ttFont):
     else:
         yield PASS, ("Glyphs do not contain duplicate components which have"
                      " the same x,y coordinates.")
+
+@check(
+    id = 'com.google.fonts/check/glyf_nested_components',
+    rationale = """
+        There have been bugs rendering variable fonts with nested components.
+        Additionally, some static fonts with nested components have been
+        reported to have rendering and printing issues. (See
+        googlefonts/fontbakery#2961 and arrowtype/recursive#412.)
```

Comment:
Do not break lines manually. The program will do it for us in the best manner for each different output such as markdown, html, text terminal, etc... Just use very long lines. Only insert line breaks at the end of your paragraphs.